ClosedXML_Tests/Excel/Cells/XLCellTests.cs

```diff
@@ -637,5 +637,15 @@ public void FormulaWithCircularReferenceFails()
                 });
             }
         }
+
+        public void FormulaWithCircularReferenceFails2()
+        {
+            Assert.Throws<InvalidOperationException>(() =>
+            {
+                var cell = new XLWorkbook().Worksheets.Add("Sheet1").FirstCell();
```

Comment:
I'd prefer to have arrangement separated from assertion. For instance, if at some point in future ```cell.FormulaA1 = "A1"``` will start throwing the exception it may remain unnoticed.

Reply:
I don't understand.

Reply:
```
            var cell = new XLWorkbook().Worksheets.Add("Sheet1").FirstCell();
            cell.FormulaA1 = "A1";
            Assert.Throws<InvalidOperationException>(() =>
            {
                var _ = cell.Value;
            });```

Like this. Here we may be absolutely sure at which step the exception is expected to be thrown.

Reply:
Oh, I see. Yes, that's better.